repo/packages/C/concord/0/package.json

```diff
@@ -6,6 +6,7 @@
   "description": "High Performance Stream Processing with C++ and Apache Mesos",
   "selected": false,
   "framework": true,
+  "hasKnownIssues": true,
```

Comment:
This PR should only change packages v3 and above. v2 packages do not support new fields.